concourse/pipelines/templates/external-table-tpl.yml

```
@@ -54,16 +54,16 @@ resources:
     repository: gcr.io/data-gpdb-ud/gpdb-pxf-dev/gpdb6-ubuntu18.04-test-pxf-hdp2
     tag: latest
     username: _json_key
-    password: ((data-gpdb-ud-google-json-key))
+    password: ((ud/pxf/secrets/google-project-id-ccp-ci-service-account-key))
 {% endif %}
 
 {% for version in range(num_versions) %}
 - name: gpdb_rhel6_rpm_latest-[[ version ]]
   type: gcs
   icon: google-drive
   source:
-    bucket: data-gpdb-ud-pivnet-artifacts
-    json_key: ((data-gpdb-ud-google-json-key))
+    bucket: ((ud/pxf/common/pivnet-artifacts-bucket-name))
+    json_key: ((ud/pxf/secrets/google-project-id-ccp-ci-service-account-key))
```

```suggestion
    json_key: ((ud/pxf/secrets/pxf-storage-service-account-key))
```

`pxf-storage-service-account` should be used since this cred is being used to read from a GCS bucket.